apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1.PopulateProductSearchTestSet/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Storage.V1;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Google.Cloud.Vision.V1.PopulateProductSearchTestSet
+{
+    /// <summary>
+    /// Application to populate a sample set of products for tests and snippets.
+    /// TODO: Handle the set (and maybe the bucket) already existing.
```

Comment:
And when this TODO is done, maybe move this code to the test projects themselves, in a fixture or test init of some kind?